README.md

```diff
@@ -1,239 +1,126 @@
 
-## Introduction
+[<img src="https://img.shields.io/badge/chat-on--slack-blue">](http://slack.overture.bio) 
+[<img src="https://img.shields.io/badge/License-gpl--v3.0-blue">](https://github.com/overture-stack/ego/blob/develop/LICENSE)
 
-[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+## Worry Less Science More
 
-Authorization Service built to provide Single Sign On for various microservices in an application. EGO works with Identity Providers such as Google, Facebook to provide social logins in the application. EGO provides stateless authorization using [JWT (JSON Web Tokens)](https://jwt.io/) and can scale very well to a large number of users.
+In this Overture repository, we (the [OICR Genome Informatics Team](https://softeng.oicr.on.ca/team/)) develop [Ego](https://www.overture.bio/products/ego/), our open-source authentication and authorization microservice. [Overture](https://www.overture.bio/) is an ensemble of modular solutions for big-data genomic science. Our core products work in concert to manage, explore and visualize molecular and clinical data. Visit our [website](https://www.overture.bio/) for more information on what Overture offers, and check out our other projects on [GitHub](https://github.com/overture-stack/).
 
-Interactive documentation of the API is provided using Swagger UI.
+## Ego - Secure Authentication & Authorization
 
-When run locally this can be found at: [http://localhost:8081/swagger-ui.html](http://localhost:8081/swagger-ui.html)
+<p align="center"><img alt="Ego in action" src="https://www.overture.bio/static/screenshot-21fc2cfc0ac1c3fd9bd7e62196477554.png"></p>
 
-<p align="center">
-    <img alt="arch" title="EGO Architecture" src="/docs/ego-arch.png" width="480">
-</p>
-<p align="center">
-  EGO Architecture
-</p>
+Access to sensitive and valuable information necessitates complex and secure methods to verify users and authorize what data and applications they are allowed to access. [Ego](https://www.overture.bio/products/ego/)  simplifies user management by providing a secure system to authenticate and authorize users of your application. [Ego](https://www.overture.bio/products/ego/) uses well-known single-sign-on identity providers like Google, GitHub, LinkedIn and ORCiD in place of managing usernames and passwords. 
 
-## Features
+Ego is [OAuth 2.0](https://oauth.net/2/) and [OpenID Connect](https://auth0.com/docs/authenticate/protocols/openid-connect-protocol) compliant. It is written in JAVA and uses [Sprint Boot](https://spring.io/projects/spring-boot) and [Spring Security Frameworks](https://spring.io/projects/spring-security). Because it uses [JSON Web Tokens (JWT)](https://jwt.io/) for stateless authorization, it can scale to accommodate many users. See our thorough [documentation](https://www.overture.bio/documentation/ego/) for further details. 
 
-Here are some of the features of EGO:
+## Related Products 
```

Comment:
Can we add a section in Related Products about Ego UI (and move the screen shot there?)

Reply:
Got it!